bubblesub/api/video.py

```diff
@@ -24,11 +24,11 @@
 import typing as T
 from pathlib import Path
 
-import ffms2
 import numpy as np
 import PIL.Image
 from PyQt5 import QtCore
 
+import ffms2
```

Comment:
This is third party, I can understand that `isort` detect this incorrectly.
Let's add an exception here: https://github.com/rr-/bubblesub/blob/master/pyproject.toml#L6 and move it back.

Reply:
Fixed manually, for now. I agree with the addition of an exception